activities/tests/shortcuts.py

```diff
@@ -13,3 +14,19 @@ def create_activity(activity_name: str, days_delta: int, people: int = 0, max_pe
     activity.people = people
     activity.save()
     return activity
+
+
+def activity_to_json(activity: models.Activity, use_can_join: bool = False):
+    """Return dict that replicates json thats contain activity data."""
+    formatted_date = activity.date.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z'
+    output = {
+        "id": activity.id,
+        "name": activity.name,
+        "detail": activity.detail,
+        "date": formatted_date,
+        "max_people": activity.max_people,
+        "people": activity.people
+    }
+    if (use_can_join):
```

Comment:
Since activity model has a method can_join() why don't you just use that method instead of get it from parameter?

Reply:
We also use the can_join method but use_can_join means "Do you want to use nenefit from this method to check something?" (Sometimes you might don't want to use it).
output['can_join'] = activity.can_join() //this can be False or True depends on method output not from use_can_join

Reply:
Ok got it